src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackLogExporter.cs

```diff
@@ -21,12 +21,11 @@
 using System.Runtime.InteropServices;
 using System.Threading;
 using Microsoft.Extensions.Logging;
-using OpenTelemetry.Internal;
 using OpenTelemetry.Logs;
 
 namespace OpenTelemetry.Exporter.Geneva;
 
-public class GenevaLogExporter : GenevaBaseExporter<LogRecord>
+internal sealed class MsgPackLogExporter : IDisposable
```

Comment:
Curious - why isn't this derived from `GenevaBaseExporter<LogRecord>`?

Reply:
There is no strong reason. We could do that as well. I don't see any major benfits to doing that either. In fact, I think we could move all of those internal fields and methods of `GenevBaseExporter` to a new abstract class called `MsgPackExporter` as all of those internal data is related to to MsgPack approach.

Reply:
Got it, makes sense 👍